### PR TITLE
Garnett is running out of time

### PR DESCRIPTION
**Pull request by superfrank**
Updated Meta styling so that it's no longer flex. Time sits in a different place. Twitter handle and email style changed so they're less shouty.

time

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
| Time sits in a different place

giphy

File: common/app/views/fragments/contentMeta.scala.html [hunk 1]
```diff
@@ -41,15 +41,16 @@
             }
             @fragments.meta.contactAuthor(item.tags, amp)
         }
+
+        @if(!(item.trail.shouldHidePublicationDate || item.content.isGallery)) {
```
Comment (regiskuckaertz) [inline common/app/views/fragments/contentMeta.scala.html]: This change will apply to everyone, are you sure about that or is it Garnett-specific?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 2]
```diff
@@ -176,6 +169,29 @@ $quote-mark: 35px;
         }
     }
 
+    .meta__contact-wrap {
+        float: left;
+        max-width: 230px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_types.scss]: Is there a grid value we could use here? like `gs-span(...)`

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 3]
```diff
@@ -203,11 +219,11 @@ $quote-mark: 35px;
 
     .byline-img {
         max-width: 60px;
-        order: 1;
+        float: left;
         border-top: 0;
-        flex: 1 100%;
         margin: 3px 0 $gs-baseline $gs-gutter/2;
         @include mq(mobileLandscape) {
+            max-width: 64px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_types.scss]: ditto

File: common/app/views/fragments/contentMeta.scala.html [hunk 4]
```diff
@@ -41,13 +41,20 @@
             }
             @fragments.meta.contactAuthor(item.tags, amp)
         }
+
+        @if((item.trail.shouldHidePublicationDate || item.content.isGallery)) {
```
Comment (regiskuckaertz) [inline common/app/views/fragments/contentMeta.scala.html]: This fragment should rather be:
```scala
@if(ActiveExperiments.isParticipating(experiments.Garnett)) {
  @if(!(item.trail.shouldHidePublicationDate || item.content.isGallery)) {
    @fragments.meta.dateline(item.trail.webPublicationDate, item.fields.lastModified, item.content.hasBeenModified, item.fields.firstPublicationDate, item.tags.isLiveBlog, item.fields.isLive)
  }
}
```

Comment (superfrank) [reply common/app/views/fragments/contentMeta.scala.html]: aha, thanks. I need to import "@import experiments._" at the top too right?

Comment (regiskuckaertz) [reply common/app/views/fragments/contentMeta.scala.html]: oh yeah sorry, totally forgot that.

**Comment by superfrank** [comment]
@regiskuckaertz Are you OK to quickly review now? Think I'm done with your changes.

**Review by regiskuckaertz — APPROVED** [review]
I don't know what's up, down, after, before anymore 😲

**Comment by superfrank** [comment]
I know what's up:

image

**Pull request merged**